src/main/backoffice/src/panels/beaconSummaryPanel/BeaconSummaryViewing.tsx

```diff
@@ -74,6 +74,10 @@ export const BeaconSummaryViewing: FunctionComponent<{
       key: "Last modified date",
       value: beacon?.lastModifiedDate,
     },
+    {
+      key: "Reference",
+      value: beacon?.referenceNumber,
```

Comment:
What would happen if `beacon` didn't have a `referenceNumber`

Reply:
Its an non null field. The ui also checks for nulls and shows N/A.